<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QResultExprDlg_Data</class>
 <widget class="QDialog" name="QResultExprDlg_Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="m_layoutTop">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="m_layoutGrid">
     <item>
      <widget class="QTabWidget" name="m_tabWidget">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>数值</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLineEdit" name="m_editValue"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>数据集</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTreeWidget" name="m_treeDataset">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>数据集</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>系统参数</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2"/>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>全局参数</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3"/>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>报告数据</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="m_btnChangeReport">
             <property name="text">
              <string>切换报告</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="m_txtPathLabel">
             <property name="text">
              <string>路径:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="m_txtPath">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="m_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_btnOK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
